.github/workflows/build.yml

```diff
@@ -627,7 +627,7 @@ jobs:
 
   typescript-tracing-tests:
     if: >
-      (github.event_name == 'pull_request' && !github.event.pull_request.head.repo.fork) || 
```

Comment:
This change is not doing anything wand forces reviews from the perm group.

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3198/commits/034979bd9a60caf2d52a33252543b0cb0b28aa30